components/interface/VFBGraph/VFBGraph.js

```diff
@@ -4,6 +4,8 @@ import GeppettoGraphVisualization from 'geppetto-client/js/components/interface/
 import CircularProgress from '@material-ui/core/CircularProgress';
 import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
+import Tooltip from '@material-ui/core/Tooltip';
+import { connect } from "react-redux";
```

Comment:
it would be good to have an external component connector as we do for the others but not of a big deal

Reply:
I left this one as is for now. Maybe I can revisit this when there's more components connecting to the redux store?